### DISCUSSION TITLE
Get Email Headers

### DISCUSSION THREAD

**Discussion by tamili2i** [ecederstrand/exchangelib#1238] [Q&A] [answered]
How to get email headers using exchangelib? I used headers key in item class but it always returns None. Or how can i get raw mail content? from that i will extract the headers.

**Comment by ecederstrand** [comment, marked as answer]
Headers are available in the `Message.headers` field. MIME content is available in the `Message.mime_content` field.

**Reply by tamili2i** [reply]
Thank you I am able to get the headers. My bad i was getting headers after filtering by only
